plugin/db/mysql/restore.go

```diff
@@ -164,17 +164,18 @@ func (driver *Driver) replayBinlog(ctx context.Context, originalDatabase, pitrDa
 		zap.String("mysqlbinlog", mysqlbinlogCmd.String()),
 		zap.String("mysql", mysqlCmd.String()))
 
-	mysqlRead, err := mysqlbinlogCmd.StdoutPipe()
+	mysqlbinlogOutput, err := mysqlbinlogCmd.StdoutPipe()
```

Comment:
Are they separate changes? Also, didn't see how the counting reader will be used.

Reply:
Added code to update replay binlog progress.
I'll update the task progress in another PR using the mechanism implemented here.